src/arch/src/aarch64/regs.rs

```diff
@@ -48,6 +51,7 @@ impl fmt::Display for Error {
             }
             self::Error::SetMP(ref e) => write!(f, "Failed to set multiprocessor state: {}", e),
             self::Error::SetSysRegister(ref e) => write!(f, "Failed to set system register: {}", e),
+            self::Error::GetMidrEl1(ref e) => write!(f, "Failed to get MIDR_EL1: {}", e),
```

Comment:
```suggestion
            self::Error::GetMidrEl1(ref e) => write!(f, "{}", e),
```
for avoiding duplicating "Failed to get MIDR_EL1" message.